components/Button.js

```diff
@@ -29,23 +30,44 @@ const Button = (props) => {
     children,
     className = "",
     loading = false,
+    tooltip = null,
     ...otherProps
   } = props;
 
   const { theme } = useTheme();
 
+  let tooltipId = null;
+
+  if (tooltip) {
+    tooltipId = `${label.replace(" ", "")}-toolTip`;
```

Comment:
```suggestion
    tooltipId = `${tooltip.replace(" ", "")}-toolTip`;
```
Should this be the tooltip value?

Reply:
This value does not really matter as long as it is consistent between the `ReactTooltip` `id` prop and the `Button` `data-for` prop